executor/tiflash_test.go

```diff
@@ -129,6 +132,9 @@ func (s *tiflashTestSuite) TestReadUnsigedPK(c *C) {
 	tk.MustExec("set @@session.tidb_isolation_read_engines=\"tiflash\"")
 	tk.MustExec("set @@session.tidb_allow_mpp=ON")
```

Comment:
enforce_mpp?

Reply:
`enforce_mpp` is not supported in release-5.0